### PR TITLE
Set injector external Vault addr protocol #1413

### PR DESCRIPTION
**Pull request by ar3ndt**

### CODE REVIEW

**Comment by mkyc** [comment]
@ar3ndt can you link related issue?

File: core/src/epicli/data/common/ansible/playbooks/roles/vault/files/vault/configure-vault.sh [hunk 1]
```diff
@@ -217,6 +217,9 @@ function configure_kubernetes {
     elif [ "${command_result[1]}" = "1" ] ; then
         log_and_print "Installing Vault Agent Helm Chart...";
         if [ "$helm_custom_values_set_bool" = "true" ] ; then
+          if [ "$vault_protocol" = "http" ] ; then
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/vault/files/vault/configure-vault.sh:220]: what if https?

Comment (ar3ndt) [reply core/src/epicli/data/common/ansible/playbooks/roles/vault/files/vault/configure-vault.sh:220]: wrong assumption, fixed now

**Comment by ar3ndt** [comment]
closing, Pawel will reconfigure a bit and include this part as well.

**Comment by mkyc** [comment]
So I'm unlinking issue. @erzetpe and @ar3ndt please link it later to correct PR.

**Pull request closed without merging**